extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/PrometheusEmitterConfig.java

```diff
@@ -103,6 +111,8 @@ public PrometheusEmitterConfig(
     this.addHostAsLabel = addHostAsLabel;
     this.addServiceAsLabel = addServiceAsLabel;
     this.extraLabels = extraLabels != null ? extraLabels : Collections.emptyMap();
+    this.pushGatewayDeleteOnShutdown = pushGatewayDeleteOnShutdown != null && pushGatewayDeleteOnShutdown;
+    this.waitForShutdownDelay = waitForShutdownDelay != null ? waitForShutdownDelay : 0;
```

Comment:
Perhaps validate that `waitForShutdownDelay` is a positive integer, if specified, and throw an invalid input exception for the operator persona if not.